cpp/tests/utilities_tests/multibuffer_memset_tests.cpp

```diff
@@ -0,0 +1,115 @@
+#include <cudf_test/base_fixture.hpp>
+#include <cudf_test/cudf_gtest.hpp>
+#include <cudf_test/type_lists.hpp>
+#include <src/io/utilities/multibuffer_memset.hpp>
+#include <thrust/iterator/transform_iterator.h>
+#include <type_traits>
+#include <cudf/io/parquet.hpp>
+
+template <typename T>
+struct MultiBufferTestIntegral : public cudf::test::BaseFixture {};
+
+TEST(MultiBufferTestIntegral, BasicTest)
+{
+    long NUM_BUFS = 1;
```

Comment:
```suggestion
```
No need for this. `BUF_SIZES.size()` tells us everything we need. Same for all the others below.